access/src/main/webapp/WEB-INF/jsp/fullRecord/neighborList.jsp

```diff
@@ -29,7 +29,12 @@
 			<c:set var="currentItemClass" scope="page">
 				<c:if test="${briefObject.id == neighbor.id}"> current_item</c:if>
 			</c:set>
-			<div class="relateditem ${currentItemClass}">
+			<c:choose>
+				<c:when test="${markedForDeletion}">
```

Comment:
Wouldn't this mark all the neighbors as deleted if the current record is deleted, even if they aren't deleted?